umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/provisioning.md

```diff
@@ -1,8 +1,49 @@
 ---
 title: Provisioning
 menuTitle: Provisioning
-description:
+description: |
+  Discover how to provision both the Data and the Device & Container
+  Infrastructures using the Management Console.
 weight: 1000
 edition: community
-draft: true
+draft: false
 ---
+
+The Management Console simplifies the deployment of the
+[Data Infrastructure](/docs/architecture/data-infrastructure/) on any existing
+system, and also guides you through the provisioning of the
+[Device & Container Infrastructure](/docs/architecture/device--container-infrastructure/).
+
+## When should I use it?
+
+Utilize the Management Console for initial deployment of the United
+Manufacturing Hub. It supports provisioning on both centralized servers and
+edge devices, offering flexibility in deployment options.
+
+## What can I do with it?
+
+Provision the United Manufacturing Hub on any existing system, and link it to
+the Management Console. The Management Console will then be able to manage the
+system and its resources.
+
```

Comment:
- install Kubernetes / k3s, the Helm Chart, etc. on a Operating System (currnetly only RHEL and flatcar are supported)
- install the default Helm Chart
- works on bare metal (if you have a x86 system, ARM is not yet supported), Virtual Machines (we use it internally on Proxmox regularly and have a guide for that), etc.
- via the Management Console also give the instance a certain name